extra/2-piping.js

```diff
@@ -16,26 +16,35 @@
   the final result to the variable goodCode
 */
 
-function add() {
+function add(num1 , num2) {
+  return num1 + num2;
 
 }
 
-function multiply() {
+function multiply(num1 , num2) {
+  return num1 * num2;
 
 }
 
-function format() {
+function format(num) {
+  return `£${num}`;
 
 }
 
+
 const startingValue = 2;
+  
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = format(multiply(add(startingValue, 10), 2));
+// MY COMMENT: This practice is not good as everything is in one line and hard to understand. 
```

Comment:
I agree! It is still good code in that you are using named functions to communicate what is happening. An example of really bad code would be to not use functions, e.g.
```
let badCode = '£' + ((startingValue + 10) * 2);
```